drivers/hmis_external_apis/app/views/hmis_external_apis/external_forms/tchc/_helpline_shelters.haml

```diff
@@ -0,0 +1,16 @@
+= render_dependent_block(input_name: 'household_type', input_value: 'single_adult') do
+  %ul.mb-0.pl-3
```

Comment:
I don't think its a good idea to keep these around in the codebase. If only for the sake of like exposing these phone numbers to bots crawling github. Maybe you can upload them onto the helpline PT story instead so we can reference them for future helpline work?

Reply:
Sure thing! All this info was taken from the public facing forms so I it's not disclosing anything. It's cleaner to remove it and archive in PT though